src/qml/qgismobileapp.qml

```diff
@@ -489,6 +489,15 @@ ApplicationWindow {
     anchors.margins: 10 * dp
   }
 
+  DropShadow {
+    visible: locatorItem.searching
+    anchors.fill: locatorItem
+    radius: 8 * dp
+    samples: 17
+    color: "#80000000"
+    source: locatorItem
+  }
+
```

Comment:
I am not a huge fan, it gets blurry, mainly on the results since the shadow has the same color as the frame.
Also, we should have coherent items: either everything has shadow (buttons, etc.) or nothing. Having only the line edit doesn't look right.

Reply:
> I am not a huge fan, it gets blurry, mainly on the results since the shadow has the same color as the frame.

As said, I couldn't test with results, I was expecting it to cover everything including result list (like e.g. on google.com)

We already have some shadows in place (on the sidebars), not sure we need to do it on everything, it was more meant to give "active" things a bit of focus.